src/posix/platform/radio_spinel.cpp

```diff
@@ -1073,6 +1112,18 @@ void RadioSpinel::RadioTransmit(void)
                     mTransmitFrame->mInfo.mTxInfo.mMaxCsmaBackoffs, mTransmitFrame->mInfo.mTxInfo.mMaxFrameRetries,
                     mTransmitFrame->mInfo.mTxInfo.mCsmaCaEnabled);
 
```

Comment:
Right before this, we call `Request(true, VALUE_SET, STREAM_RAW, ...)`. IIRC, the `true` is `aWait` param indicating we should send request and wait for response to be received. If that's the case, wouldn't we already have the tx done signal before we get back here?

Reply:
I checked the code and figured it out. We handle the `STREAM_RAW` property as  special case in `RequstV,